package/health_daemon/src/baseline_monitor.c

```diff
@@ -26,89 +26,94 @@
 
 /* From libswiftnav-private - These really belong in libsbp */
 /* Define Position types - currently aligned to SBP messages */
-#define NO_POSITION         0
-#define SPP_POSITION        1
-#define DGNSS_POSITION      2
-#define FLOAT_POSITION      3
-#define FIXED_POSITION      4
-#define POSITION_MODE_MASK  0x07 /* Bits 0-2 */
-
-#define BASELINE_THRESHOLD (80000.0f) /* m */
+#define NO_POSITION 0
+#define SPP_POSITION 1
+#define DGNSS_POSITION 2
+#define FLOAT_POSITION 3
+#define FIXED_POSITION 4
+#define POSITION_MODE_MASK 0x07 /* Bits 0-2 */
+
+#define BASELINE_THRESHOLD (80000.0f)     /* m */
 #define BASELINE_ALERT_RATE_LIMIT (1000u) /* ms */
 
 #define MM_TO_M_FLOAT(val_in_mm) ((float)(val_in_mm) / 1000)
-static health_monitor_t* baseline_monitor;
+static health_monitor_t *baseline_monitor;
 
-struct baseline_monitor_ctx_s {
+static struct baseline_monitor_ctx_s {
   bool past_threshold;
   float distance_over_threshold;
-} baseline_monitor_ctx = {
-  false,
-  0.0
-};
+} baseline_monitor_ctx = { false, 0.0 };
 
-static int sbp_msg_baseline_ecef_callback(health_monitor_t* monitor, u16 sender_id, u8 len, u8 msg_[], void *ctx)
+static int sbp_msg_baseline_ecef_callback(health_monitor_t *monitor,
+                                          u16 sender_id,
+                                          u8 len,
+                                          u8 msg_[],
+                                          void *ctx)
 {
   int result = 0;
   (void)monitor;
   (void)sender_id;
   (void)len;
-  msg_baseline_ecef_t *msg = (void*)msg_;
+  msg_baseline_ecef_t *msg = (void *)msg_;
```

Comment:
Curious - doing a case here to a `(void*)` instead of to `msg_baseline_ecef_t*` isn't it then doing an implicit cast to `msg_baseline_ecef_t*`?

Reply:
Good catch. Fixed and merged.